### PR TITLE
Ignite .NET SDK released

### PR DESCRIPTION
**Pull request by MikhailTryakhov**
## Description
Built SDK for swagger September branch https://github.com/Azure/azure-rest-api-specs/ for Azure NW

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by MikhailTryakhov** [comment]
Is this still 19.0.2 right? 
https://github.com/Azure/azure-sdk-for-net/compare/psSdkJson6...MikhailTryakhov:local-september#diff-6bab6934b59abbd5fd615ec26e5f6b6dR10

File: src/SDKs/Network/Management.Network/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Network Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure Network management functions for managing the Microsoft Azure Network service.")]
 
-[assembly: AssemblyVersion("19.0.2.0")]
```
Comment (MikhailTryakhov) [inline src/SDKs/Network/Management.Network/Properties/AssemblyInfo.cs]: @dsgouda  still 19.0.2 right?

**Comment by MikhailTryakhov** [comment]
@dsgouda  [assembly: AssemblyVersion("19.0.2.0")] is still the same right?

File: src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj [hunk 2]
```diff
@@ -7,7 +7,7 @@
 		<PackageId>Microsoft.Azure.Management.Network</PackageId>
 		<Description>Provides management capabilities for Network services.</Description>
 		<AssemblyName>Microsoft.Azure.Management.Network</AssemblyName>
-		<Version>19.3.0-preview</Version>
+		<Version>19.4.0-preview</Version>
 		<PackageTags>Microsoft Azure Network management;Network;Network management;</PackageTags>    
 		<PackageReleaseNotes>
 		<![CDATA[
```
Comment (dsgouda) [inline src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj]: Please update `PackageReleaseNotes`

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks good apart from a minor comment

**Review by dsgouda — APPROVED** [review]
LGTM subject to CIs passing

**Review by dsgouda — CHANGES_REQUESTED** [review]
Comments need to be addressed

**Review by dsgouda — APPROVED** [review]
LGTM

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**